plugins/gradle/graphql-java-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphQLCodegenGradleTask.java

```diff
@@ -73,6 +75,8 @@ public void generate() throws Exception {
         MappingConfig mappingConfig = new MappingConfig();
         mappingConfig.setPackageName(packageName);
         mappingConfig.setCustomTypesMapping(customTypesMapping);
+        mappingConfig.setApiNamePrefix(apiNameSuffix);
```

Comment:
Looks like there is the same prefix/suffix mistake here

Reply:
Thanks! Fixed